dts/bindings/input/pixart,pat912x.yaml

```diff
@@ -25,3 +25,35 @@ properties:
     description: |
       The input code for the Y axis to report for the device, typically any of
       INPUT_REL_*. No report produced for the device Y axis if unspecified.
+
+  res-x-cpi:
+    type: int
+    description: |
+      CPI resolution for the X axis, range 0 to 1275, rounded down to the
+      closest supported value in increments of 5.
+
+  res-y-cpi:
+    type: int
+    description: |
+      CPI resolution for the Y axis, range 0 to 1275, rounded down to the
+      closest supported value in increments of 5.
+
+  invert-x:
+    type: boolean
+    description: |
+      Invert X axis values.
+
+  invert-y:
+    type: boolean
+    description: |
+      Invert Y axis values.
+
+  sleep1-enable:
+    type: boolean
+    description: |
+      Enable sleep1 mode.
+
+  sleep2-enable:
+    type: boolean
+    description: |
+      Enable sleep2 mode, only valid if sleep1 is also enabled.
```

Comment:
Maybe enforce this with a build_check?

Reply:
You are exactly right, got lazy and figure I'd just get away with runtime check. :-) Thanks for keeping me honest.

Reply:
Oh I assumed this was about the cpi range, let me add one for sleep as well